ampd/src/lib.rs

```diff
@@ -20,7 +21,7 @@ use tofnd::grpc::{MultisigClient, SharableEcdsaClient};
 use types::TMAddress;
 
 use crate::config::Config;
-use crate::error::Error;
```

Comment:
But this mod is still there?

Reply:
what are you asking? The error cases have been merged into the lib mod (because they are the errors of this mod) and `LoggableError` and reporting extensions have been moved into the `packages` folder for reuse in contracts in a different PR